core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/form/ContainerImpl.java

```diff
@@ -129,10 +111,6 @@ private void initModel() {
                 redirect = contextPath + redirect;
             }
         }
-
-        if (!StringUtils.equals(request.getRequestPathInfo().getExtension(), ExporterConstants.SLING_MODEL_EXTENSION)) {
```

Comment:
I think I accidentally removed this thinking it had to do with sling model exporting, but it looks like this code is supposed to run if *not* doing sling model export, so I wonder if we should restore `runActionTypeInit(formStructureHelper);` without the `if` statement

Reply:
Restored this functionality. `ExporterConstants.SLING_MODEL_EXTENSION` does not exist in 6.2 so I checked what its value is from https://helpx.adobe.com/experience-manager/6-4/sites/developing/using/reference-materials/javadoc/constant-values.html and hardcoded `"json"`